### PR TITLE
Improved error handling by adding IssuerNotTrusted exception

### PR DESCRIPTION
**Pull request by irinaepshteyn**

### CODE REVIEW

**Comment by irinaepshteyn** [comment]
All tests PASSED: https://predix1.jenkins.build.ge.com/job/Predix-Security/job/token-lib/job/token-lib-dev-pullrequest/117/

File: src/main/java/com/ge/predix/uaa/token/lib/exceptions/IssuerNotTrustedException.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    public IssuerNotTrustedException(final String msg) {
+        super(msg);
+   }
+
+    public IssuerNotTrustedException(final String msg, final Throwable t) {
+        super(msg, t);
+    }
+
+    public String getOAuth2ErrorCode() {
```
Comment (anubhavi25) [inline src/main/java/com/ge/predix/uaa/token/lib/exceptions/IssuerNotTrustedException.java]: Add the `@Override` annotation above this method signature

File: src/main/java/com/ge/predix/uaa/token/lib/exceptions/IssuerNotTrustedException.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.security.oauth2.common.exceptions.InvalidTokenException;
+
+@SuppressWarnings("serial")
+public class IssuerNotTrustedException extends InvalidTokenException {
+
+    public IssuerNotTrustedException(final String msg) {
+        super(msg);
+   }
+
+    public IssuerNotTrustedException(final String msg, final Throwable t) {
```
Comment (anubhavi25) [inline src/main/java/com/ge/predix/uaa/token/lib/exceptions/IssuerNotTrustedException.java]: This constructor is never used so we should remove it.

**Comment by sanjeevchopra** [comment]
looks good, no additional comments.

**Comment by irinaepshteyn** [comment]
All tests PASSED: https://predix1.jenkins.build.ge.com/job/Predix-Security/job/token-lib/job/token-lib-dev-pullrequest/118/

**Pull request merged**